packages/cli/commands/import/workflow.ts

```diff
@@ -27,51 +45,30 @@ export class ImportWorkflowsCommand extends Command {
 		separate: flags.boolean({
 			description: 'Imports *.json files from directory provided by --input',
 		}),
+		id: flags.string({
```

Comment:
`id` is very general would suggest `owner`, `user` or the respective `ownerId/userId`
```suggestion
		owner: flags.string({
```